src/ui/components/zero-state/template.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="zero-state">
+  <h1>
+    The framework for creating <strong>ambitious</strong> web applications
+    also should be <strong>updated</strong>
```

Comment:
`Sometimes the framework for creating <strong>ambitious</strong> web applications needs to be <strong>updated</strong>`